### ISSUE TITLE
HC Renewal Script error message

### ISSUE THREAD

**Issue by KKohlmeyer** [MN-Script-Team/DHS-MAXIS-Scripts#2472]
When running the HC Renewal Script, we get an error message that states, "Script Error at Line 1, Column1:Type mismatch: 'cint'

Multiple workers have gotten this.  Thank you!

**Comment by CDPotter** [comment]
Does it seem to be ending on any MAXIS panel in particular? I'm assuming this occurs while it's trying to gather data from all of the panels?

**Comment by KKohlmeyer** [comment]
It run through all the panels and stops on the SELF panel.

Karen Kohlmeyer
Houston County Human Services
Child Support/Financial Assistance Supervisor

Phone 507-725-5893
Fax 507-725-3990

Karen.kohlmeyer@co.houston.mn.us

**Comment by CDPotter** [comment]
Hmm in testing it here on Release and on Master we were unable to replicate any CINT error. Could you try checking your API settings against what is listed here?

"3.	On the menu bar, click “Options”, then “API”. Make sure “Enable DDE Server Interface”, “Auto-Assign HLLAPI Names”, and “Allow Multiple Simultaneous Connections” are all checked. Uncheck “Auto-Launch the BlueZone DOS HLLAPI Redirector”. Press “OK” when finished."

**Comment by KKohlmeyer** [comment]
That is what I have.  I just tried it again too and it works in 08/16 footer month, but the error happens in 06/16.  Maybe it just doesn’t like going backwards.  Now that I am looking, the REIN programs script may have been the better choice.  I’m not in scripts all that often, so think this was a user error.  We are having EDMS issues, so I’ve been doing more processing than usual!  ☺

Thank you for the help!!!

Karen Kohlmeyer
Houston County Human Services
Child Support/Financial Assistance Supervisor

Phone 507-725-5893
Fax 507-725-3990

Karen.kohlmeyer@co.houston.mn.us

**Comment by CDPotter** [comment]
Aha the 06/16 is exactly what did it! I'll look into it and see what I can try to figure out today. Thanks!

**Comment by CDPotter** [comment]
Ok after some digging I found the error. We added code to check the GRH PIC, in doing so the script is told to PF3 to get out of the GRH PIC. However the GRH PIC doesn't exist 06/16 and before as such the script leaves the JOBS panel and tried to convert a letter if finds on the self panel into a integer (CINT).

This is a funclib fix so I will create and issue there and submit a fix. Thank you for reporting!
